Source/MethodCall.cs

```diff
@@ -416,51 +385,67 @@ public IVerifies AtMost(int callCount)
 		protected IVerifies RaisesImpl<TMock>(Action<TMock> eventExpression, Delegate func)
 			where TMock : class
 		{
-			var ev = eventExpression.GetEvent((TMock)Mock.Object);
-			this.mockEvent = ev.MemberInfo;
-			this.mockEventArgsFunc = func;
+			var ev = eventExpression.GetEvent((TMock)mock.Object).MemberInfo;
+			if (ev != null)
+			{
+				this.raiseEventResponse = new RaiseEventResponse(this.mock, ev, func, null);
+			}
+			else
+			{
+				this.raiseEventResponse = null;
```

Comment:
This should possibly `throw` instead, like `Callback` does when given a `null` argument.